scss/_grid.scss

```diff
@@ -10,7 +10,7 @@ Mobile First - This applies from 0px to 767px
 
 /***** Row setup *****/
 .row                                            { margin:0 auto; padding:0 20px; display:flex; flex-direction:row; box-sizing:border-box; flex:0 1 auto; flex-wrap:wrap; }
-.row.row--width-fluid                           { width:100%; }
+.row .row--width-fluid                          { width:100%; }
```

Comment:
Are you saying that your way means you can do 

```html
<div class="row--width-fluid"></div>
```

and currently you have to do

```html
<div class="row row--width-fluid"></div>
```

If so, then sure, but without the row class, you don't get the container set to display flex which is needed for the actual grid behaviour right?